### PR TITLE
Update license file(s)

### PR DESCRIPTION
**Pull request by lance**
As part of the transition to CNCF, we now have a CLO Monitor tool which
tells us if our licenses are up to snuff. The license for this repository
is not.

https://clomonitor.io/projects/knative/knative#community_license)

This commit splits the license into two separate files, the LICENSE file
covers all code, and the LICENSE-docs file covers all docs, per CNCF
reccommendations. https://cncfservicedesk.atlassian.net/servicedesk/customer/portal/1/CNCFSD-1168

Fixes: https://github.com/knative/community/issues/970

/kind cncf-infra

Signed-off-by: Lance Ball <lball@redhat.com>

### CODE REVIEW

**Comment by lance** [comment]
> /hold @lance I'm concerned that this PR deletes the file `LICENSE` is intentional? My understanding was we were going to have 2 files `LICENSE` and `LICENSE-docs` Not having `LICENSE` affects github oss ranking

This PR does not delete the `LICENSE` file at all! :) It simply removes the Creative Commons section and puts it in a new `LICENSE-docs` file.

File: LICENSE-docs [hunk 1]
```diff
@@ -0,0 +1,390 @@
+SPDX-License-Identifier: Apache-2.0
```
Comment (csantanapr) [inline LICENSE-docs:1]: Is this a mistake?

Comment (csantanapr) [reply LICENSE-docs:1]: Should this crative commons

Comment (lance) [reply LICENSE-docs]: @csantanapr yes good catch. It should be in the main `LICENSE` file. Fixed.

**Pull request merged**